### ISSUE TITLE
Better File selector drawer

### ISSUE THREAD

**Issue by hex007** [nikita36078/J2ME-Loader#237]
Hey @nikita36078 I have a working File selector like this. Would you be interested in incorporating this or changing the current one. The current one looks outdated and I think it needs a fresher look.

![screenshot_20180323-012149](https://user-images.githubusercontent.com/1689246/37819142-b57c44c4-2e39-11e8-8531-f179097e649a.png)

**Comment by nikita36078** [comment]
Yes, it is a good idea. Yours seems to be interesting, however I need to look at another 3rd party filepickers, too :)

**Comment by nikita36078** [comment]
Well, I ended up with https://github.com/spacecowboy/NoNonsense-FilePicker, because I already used it before and it is pretty flexible.

**Comment by oatmeal01** [comment]
the new file browser is great, how about adding file size info and visible file name and file extension on the next update.

**Comment by AdityaMH** [comment]
And then, why java files with .JAR (not .jar) format not listed in file drawer?

**Comment by oatmeal01** [comment]
@aditya all file types are excluded except .jar and .jad

**Comment by AdityaMH** [comment]
why not include .JAR (capitalized format) in file drawer? My games lot with .JAR format.

**Comment by nikita36078** [comment]
@AdityaMH It was a bug, fixed now.

**Comment by AdityaMH** [comment]
And why not add  searching file, sort file, and icon jar too?

**Comment by oatmeal01** [comment]
@nikita36078 why does jad format can't be  installed outside the app(you still need to browse for it through the filepicker). Unlike jar, you can install it through any third party file manager or easily install through browser after downloading. 
Can you implement jad installation outside the app.

**Comment by woesss** [comment]
@oatmeal01 https://github.com/nikita36078/J2ME-Loader/blob/8ac59481902a9184ba0e8e372012c0920a2c4064/app/src/main/AndroidManifest.xml#L58 - JAD and JAR are supported equally. Your system does not offer to choose in which application to open JAD? She open it in another application or reports that there are no applications to open?

**Comment by oatmeal01** [comment]
> @oatmeal01
> 
> https://github.com/nikita36078/J2ME-Loader/blob/8ac59481902a9184ba0e8e372012c0920a2c4064/app/src/main/AndroidManifest.xml#L58
> 
> - JAD and JAR are supported equally. Your system does not offer to choose in which application to open JAD? She open it in another application or reports that there are no applications to open? Can you write a system log when trying to open JAD (need log by tag 'ActivityManager')?

**Installing JAD file through 3rd Party File Manager**
- Failed to install
![Screenshot_20191208_091720](https://user-images.githubusercontent.com/36059629/70382786-6ba83600-199c-11ea-9884-a86bfde497a4.png)

**Installing JAD file through J2ME Loader file manager**
- Can only install jad through J2ME Loader file manager.
![Screenshot_20191208_091858](https://user-images.githubusercontent.com/36059629/70382808-af02a480-199c-11ea-8c5c-c3cf19362a7b.png)

**Installing JAR file through 3rd Party File Manager**
- Successful Installation
![Screenshot_20191208_092154](https://user-images.githubusercontent.com/36059629/70382812-e5402400-199c-11ea-9339-f392de772d59.png)
